templates/admin/committee/list_hosts.html.twig

```diff
@@ -1,19 +1,36 @@
 {% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
 
+{% import _self as list_host %}
+
+{% macro dislay_host_line(host) %}
+    <li>
+        <a href="{{ path('admin_app_adherent_show', {'id': host.id}) }}">
```

Comment:
CS : please remove simple quotes for array keys  and add white spaces around arrays : 
`<a href="{{ path('admin_app_adherent_show', { id: host.id }) }}">`

Reply:
I always forgot it :'(